Packs/ThreatIntelligenceManagement/Scripts/IncidentsPerFeed/IncidentsPerFeed.yml

```diff
@@ -0,0 +1,37 @@
+args:
+  - default: false
+    description: Additional filters for the 'getIncidents' command
+    isArray: false
+    name: query
+    required: false
+    secret: false
+  - default: false
+    description: Name to display in the widget as the distinct incidents name
+    isArray: false
+    name: incidents_distinction_name
+    required: false
+    secret: false
+comment: "Return amount of incidents that has indicators from feeds in them.\nOptional-\
+  \ a query can be added and the script will then return the incident amount categorized\
+  \ by the query.\nMeaning it will display how much of the incidents that came from\
+  \ this feed returned from the query. "
```

Comment:
This can also be made to be clearer, but we can have @kirbles19 review it if you're unsure how it can be improved.

Reply:
I'll talk to him and see if he has any better idea